Query for exact timestamp causes panic

This query works:
`SELECT count(msgCount) FROM outbounddata WHERE providerId='1' AND time > now() - 15m`

This query causes panic: ( time = vs time > )
`SELECT count(msgCount) FROM outbounddata WHERE providerId='1' AND time = now() - 15m`

```
panic: runtime error: integer divide by zero
[signal 0x8 code=0x1 addr=0x6c318d pc=0x6c318d]

goroutine 5120 [running]:
github.com/influxdb/influxdb/influxql.(*MapReduceJob).Execute(0xc20bea8310, 0xc20be318c0, 0xc209290700)
        /root/.gvm/pkgsets/go1.4.2/global/src/github.com/influxdb/influxdb/influxql/engine.go:138 +0x68d
github.com/influxdb/influxdb/influxql.(*Executor).execute(0xc20be79440, 0xc20be318c0)
        /root/.gvm/pkgsets/go1.4.2/global/src/github.com/influxdb/influxdb/influxql/engine.go:774 +0xba
created by github.com/influxdb/influxdb/influxql.(*Executor).Execute
        /root/.gvm/pkgsets/go1.4.2/global/src/github.com/influxdb/influxdb/influxql/engine.go:753 +0x5a
```

Schema:
```
{
    "database": "streamDB",
    "retentionPolicy": "default",
    "points": [
        {
            "name": "outbounddata",
            "tags": {
                "providerId": "1",
                "customerId": "xyz"
               
            },
            "fields": {
                "stream":"stream1",
                           "subscription":"sdfsd1",
                           "msgCount": 1
            }
        }
    ]
}
```

--- a/cmd/influxd/server_integration_test.go
+++ b/cmd/influxd/server_integration_test.go
@@ -674,6 +674,12 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 			expected: `{"results":[{"series":[{"name":"cpu","columns":["time","value"],"values":[["2000-01-01T00:00:10Z",30]]}]}]}`,
 		},
 		{
+			name:     "aggregation with no interval",
+			query:    `SELECT count(value) FROM cpu WHERE time = '2000-01-01 00:00:00'`,
+			queryDb:  "%DB%",
+			expected: `{"results":[{"series":[{"name":"cpu","columns":["time","count"],"values":[["2000-01-01T00:00:00Z",2]]}]}]}`,
+		},
+		{
 			name:     "sum aggregation",
 			query:    `SELECT SUM(value) FROM cpu WHERE time >= '2000-01-01 00:00:05' AND time <= '2000-01-01T00:00:10Z' GROUP BY time(10s), region`,
 			queryDb:  "%DB%",

--- a/influxql/engine.go
+++ b/influxql/engine.go
@@ -135,7 +135,10 @@ func (m *MapReduceJob) Execute(out chan *Row, filterEmptyResults bool) {
 	resultValues := make([][]interface{}, pointCountInResult)
 
 	// ensure that the start time for the results is on the start of the window
-	startTimeBucket := m.TMin / m.interval * m.interval
+	startTimeBucket := m.TMin
+	if m.interval > 0 {
+		startTimeBucket = startTimeBucket / m.interval * m.interval
+	}
 
 	for i, _ := range resultValues {
 		var t int64

